sinks/librato.py

```diff
@@ -288,13 +285,16 @@ def add_measure(self, key, value, time):
             k = name
```

Comment:
Key is being [truncated prematurely](https://github.com/librato/statsite/blob/0b7dcc1403564f796da0b2e6782a0f479d6270d1/sinks/librato.py#L224).  Truncation should at least result in a warning, if not an error, and should happen _after_ all source and/or tags have been extracted.